### PR TITLE
 DBZ-1350 NO_AUTO_CREATE_USER unsupported on MySQL 8

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.jboss.org/browse/DBZ-1350
Contains https://github.com/debezium/debezium/pull/935

### CODE REVIEW

File: debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlLexer.g4 [hunk 1]
```diff
@@ -1167,7 +1167,7 @@ fragment CHARSET_NAME:               ARMSCII8 | ASCII | BIG5 | BINARY | CP1250
                                      | LATIN1 | LATIN2 | LATIN5 | LATIN7
                                      | MACCE | MACROMAN | SJIS | SWE7 | TIS620
                                      | UCS2 | UJIS | UTF16 | UTF16LE | UTF32
-                                     | UTF8 | UTF8MB4;
+                                     | UTF8 | UTF8MB3 | UTF8MB4;
```
Comment (gunnarmorling) [inline debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlLexer.g4]: Hum, did something get mixed up here? It's the fix from the DBZ-1349 PR?

Comment (jpechane) [reply debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlLexer.g4]: It is in the original comment

Comment (jpechane) [reply debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlLexer.g4]: @gunnarmorling Rebased to the updated master

Comment (gunnarmorling) [reply debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlLexer.g4]: Gotcha. Thx.

**Pull request merged**